tests/extmod/ussl_basic.py.exp

```diff
@@ -1,5 +1,4 @@
-ssl_handshake_status: -256
-wrap_socket: OSError(5,)
+wrap_socket: OSError(-256, 'CONN_LOST')
```

Comment:
This test obviously only works with axtls, but that's ok for now.

Reply:
As written the test can only work with axtls, which is a shame. The problem is that using io.Bytes doesn't work with mbedtls.